posthog/models/event/util.py

```diff
@@ -362,10 +362,12 @@ def get_event_count_for_team(team_id: Union[str, int]) -> int:
 
 
 def get_event_count() -> int:
+    """
+    ```SELECT count(1) as count FROM events``` is too slow on cloud
+    """
     result = sync_execute(
         """
-        SELECT count(1) as count
-        FROM events
+        SELECT sum(rows) FROM system.parts WHERE (active = 1) AND (table = 'sharded_events')
```

Comment:
Does every installation have `sharded_events` ?

Reply:
We create the table with 

```
EVENTS_DATA_TABLE_ENGINE = lambda: ReplacingMergeTree(
    "events", ver="_timestamp", replication_scheme=ReplicationScheme.SHARDED
)
```

So I think so yes

Reply:
I worried that this was a result of the change in #17525 but that query succeeds in metabase